packages/calcite-components/src/assets/styles/_floating-ui.scss

```diff
@@ -15,7 +15,8 @@ $floating-ui-transform-right: translateX(-5px);
     transition-property: transform, visibility, opacity;
     opacity: 0;
     box-shadow: $shadow-2;
```

Comment:
We should add a `--calcite-tooltip-shadow` prop for this and default it to this shadow's token equivalent.

Reply:
Whats the token equivalent? @alisonailea